tools/chplcheck/src/driver.py

```diff
@@ -59,12 +60,11 @@ class LintDriver:
     for registering new rules.
     """
 
-    def __init__(self, skip_unstable: bool, internal_prefixes: List[str]):
+    def __init__(self, config: Config):
+        self.config: Config = config
         self.SilencedRules: List[str] = []
         self.BasicRules: List[Tuple[str, Any, rule_types.BasicRule]] = []
         self.AdvancedRules: List[Tuple[str, rule_types.AdvancedRule]] = []
-        self.skip_unstable: bool = skip_unstable
-        self.internal_prefixes: List[str] = internal_prefixes
```

Comment:
I'd suggest keeping these aliases for fields in `config` for brevity.

Reply:
I would rather have less duplication, these aliases are only used once or twice